src/components/relatedItems/Card.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import axios from 'axios';
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Why so much state? Some of these never change so might be able to just be props?

Reply:
Also add a key prop to your list of cards. Its super easy to do and makes it so you don't have to keep seeing the error come up in console.

Reply:
So my cardList component will receive the current productId from App, and then will get the related items array for that productId and create a card for each related item productId. So right now, the only thing being passed into each card is the productId from the related items API call, and everything in state is what I'm getting in my getAllInfo function, which I'm then storing in state to update the card. Good point though that I could probably move all of this logic up to cardList and just pass that info down as props. I'll take a look into that.